tests/settings/ConfigurationGatheringTest.py

```diff
@@ -70,15 +70,16 @@ def test_gather_configuration(self):
 
```

Comment:
the `escape` above also has to be removed. And just like with the other `escape` removal below: No `PathArg`. Just remove